engine/queue/score-queue-impl/src/main/java/io/cloudslang/engine/queue/repositories/ExecutionQueueRepositoryImpl.java

```diff
@@ -238,6 +297,19 @@ public void init() {
         deleteFinishedStepsJdbcTemplate = new JdbcTemplate(dataSource);
         findPayloadByExecutionIdsJdbcTemplate = new JdbcTemplate(dataSource);
         getBusyWorkersJdbcTemplate = new JdbcTemplate(dataSource);
+
+        useLargeMessageQuery = StringUtils.equalsIgnoreCase("true", System.getProperty("score.poll.use.large.message.query", "true"));
```

Comment:
`parseBoolean()` instead of `StringUtils.equalsIgnoreCase()` no need for extra dependency